ui-speedspacechart/src/components/utils.ts

```diff
@@ -294,6 +295,9 @@ export const binarySearch = <T>(data: T[], element: number, lambda: (element: T)
   return left;
 };
 
+/** Convert meters to kilometers */
+export const metersToKilometers = (meters: number) => meters / 1000;
```

Comment:
In stories/utils.ts, we have some functions like this one `convertMmToKM`. Maybe we should but them all at the same place or at least harmonize the naming.

Reply:
It seems normal to me that these functions are separated. The one in stories/utils is not used by the component and should therefore be ignored when building the npm package.

Reply:
but it's used in OSRD, that's why we use the utils in the story instead of the component itself

Reply:
Ok fine by me so second part of my first comment, we could still keep the same naming convention as the ones in other utils :)

```suggestion
export const convertMToKm = (meters: number) => meters / 1000;
```